package.json

```diff
@@ -23,13 +24,15 @@
   "author": "Aldwin Vlasblom <aldwin@avaq.it> (https://github.com/Avaq)",
   "license": "MIT",
   "dependencies": {
-    "callgebra": "^1.0.0"
+    "callgebra": "^2.0.0"
   },
   "peerDependencies": {
-    "fluture": ">=11.0.0 <12.0.0"
+    "fluture": ">=12.0.0-beta.4 <13.0.0"
```

Comment:
Do we still want to support `12.0.0-beta.4`, `12.0.0-beta.5` and `12.0.0-beta.6`?

Reply:
Yeah, they work, so why not.